packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -3,7 +3,8 @@ import cx from "classnames";
 import { BaseSizes, SIZES_VALUES } from "../../constants";
 import React, { forwardRef, useCallback, useMemo, useRef, useState } from "react";
 import Select, { InputProps, components, createFilter, ActionMeta } from "react-select";
-import AsyncSelect from "react-select/async";
+// @ts-ignore
+import AsyncSelect from "react-select/async/dist/react-select.esm.js";
```

Comment:
add d.ts for react-select/async
we already have d.ts for react-select I think in packages.d.ts or something